src/routes/[type=mediaType]/[id]/+page.ts

```
@@ -22,17 +22,19 @@ export const load = (async ({ fetch, params }) => {
 					'credits,external_ids,recommendations,similar,videos,keywords',
 					id
 				);
 			case 'tv':
-				return await getTVDetails(
+				const result = await getTVDetails(
 					fetch,
-					'en-US',
+					TMDB_LANGUAGE,
 					'credits,external_ids,recommendations,similar,videos,keywords',
 					id
 				);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)